src/metatable.js

```diff
@@ -21,10 +16,12 @@ function metatable(options) {
             var sel = d3.select(this),
                 table;
 
-            var keyset = d3.set();
+            var keys = [];
```

Comment:
thoughts on using Set or continuing to use d3.set here? I used a set abstraction before because we'll repeatedly need to test for membership in the set, and `keys.indexOf` is O(n) wrt the number of keys in the set, whereas a set-like datastructure should do O(1) or at most O(nlogn)

Reply:
I used the array to let the columns stay in the same position, I had some problem with them not in the right position. But I think that was caused by the immutable selections of v4.

Reply:
I inserted the set back to gather the keys, but use the array in completeRename (and for consistency in completeDelete) now to prevent the column from jumping to the end.